modules/persistent-storage-hostpath-static-provisioning.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-vsphere.adoc
+
+[id="vsphere-static-provisioning_{context}"]
+= Statically provisioning hostPath volumes
```

Comment:
Don't we list this in https://github.com/openshift/openshift-docs/pull/17603/files#diff-1c65246cb57e25d53ab01b23a5d211de